client/src/components/Citation/CitationsList.vue

```diff
@@ -43,6 +46,57 @@ onMounted(async () => {
         console.error(e);
     }
 });
+
+function copyBibtex() {
+    let text = "";
+    citations.value.forEach((citation) => {
+        const cite = citation.cite;
+        const bibtex = cite.format("bibtex", {
+            format: "text",
+            template: "bibtex",
+            lang: "en-US",
+        });
+        text += bibtex;
+    });
```

Comment:
I would extract this to a `citationsToBibtexAsText()` function and reuse it for the download too :)

Reply:
@davelopez yeah, of course, thanks!